client/cmd/dexc/ui/depthchart.go

```diff
@@ -95,7 +98,9 @@ func (c *depthChart) calcRows() []string {
 	if err != nil {
 		// Don't update the UI from a Draw method, so we won't print an error here.
 		// TODO: Add a file-only logger for logging these errors?
-		return nil
+		app.QueueUpdate(func() {
+			log.Errorf("interpolate error: %v", err)
+		})
```

Comment:
heh, not too bad after all.  :man_facepalming:   I missed that `app.QueueUpdate` was available here.